lib/Loop/EvDriver.php

```diff
@@ -10,21 +10,34 @@
 class EvDriver extends Driver {
     /** @var \EvSignal[]|null */
     private static $activeSignals;
+
     /** @var \EvLoop */
     private $handle;
+
     /** @var \EvWatcher[] */
     private $events = [];
+
     /** @var callable */
     private $ioCallback;
+
     /** @var callable */
     private $timerCallback;
+
     /** @var callable */
     private $signalCallback;
+
     /** @var \EvSignal[] */
     private $signals = [];
 
+    /** @var int Internal timestamp for now. */
+    private $now;
+
+    /** @var bool */
+    private $nowUpdateNeeded = true;
```

Comment:
in the other loop, this defaults to `false`

Reply:
Oops, it should be the same to be consistent, though the initial value doesn't matter in this loop.